src/components/Amalgamation/BusinessTableSummary.vue

```diff
@@ -85,7 +85,7 @@ export default class BusinessTableSummary extends Vue {
   }
 
   email (item: AmalgamatingBusinessIF): string {
-    if (item?.type === AmlTypes.LEAR) return item.email
+    if (item?.type === AmlTypes.LEAR) return item.authInfo?.contacts[0]?.email
```

Comment:
Works the same as before. Example:

![image](https://github.com/bcgov/business-create-ui/assets/10002889/e4abf720-7043-45da-af37-42357fc472c7)